test/fixtures/served/amp-dynamic-css-classes.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP #0</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-dynamic-css-classes" src="/base/dist/v0/amp-dynamic-css-classes-0.1.max.js"></script>
```

Comment:
There's no element included yet, but the plan was for eventual A/B testing.

@cramforce wanted this to be in an extension and not in core AMP ("it has too much already").